docs/3.Scans/resource-scans.md

```diff
@@ -187,7 +187,7 @@ nav_order: 1
 | 176 | CKV_AWS_95   | resource          | AWS::ApiGatewayV2::Stage                                                     | Ensure API Gateway V2 has Access Logging enabled                                                                                                                                                         | Cloudformation |
 | 177 | CKV_AWS_96   | resource          | aws_rds_cluster                                                              | Ensure all data stored in Aurora is securely encrypted at rest                                                                                                                                           | Terraform      |
 | 178 | CKV_AWS_96   | resource          | AWS::RDS::DBCluster                                                          | Ensure all data stored in Aurrora is securely encrypted at rest                                                                                                                                          | Cloudformation |
-| 179 | CKV_AWS_97   | resource          | aws_ecs_task_definition                                                      | Ensure Encryption in transit is enabled for ECS Task defintion EFS volumes                                                                                                                               | Terraform      |
+| 179 | CKV_AWS_97   | resource          | aws_ecs_task_definition                                                      | Ensure Encryption in transit is enabled for ECS Task definition EFS volumes                                                                                                                               | Terraform      |
```

Comment:
Thanks @kjhosein. `resource-scans.md` is auto-generated from the classes. can you do the change in ECSTaskDefinitionEFSVolumeEncryption